frontend/src/worksheets/CreateWorkSheetForm.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import { Formik, Form } from 'formik';
+import { Link } from 'react-router-dom';
+import Input from '../Input';
+import Button from '../Button';
+import PopUp from '../PopUp';
+import SelectInput from '../SelectInput'
+import getCurrentDate from './Date';
+import schema from './ValidationWorkSheet'
+
+function CreateWorkSheetForm({ sent, setSent, sentSuccessfully, popUpMessage, sendData, path, basePath, title }) {
+
+  const itemList1 = [{ id: 1, label: "Telepítés", value: "INSTALLATION" },
```

Comment:
Szerintem az id-ne nincs értelme az egyes elemeknél... a label pont elég, sőt így akár már enum is lehetne 🤔